src/database/DataTypes/Name.js

```diff
@@ -161,6 +161,7 @@ export class Name extends Realm.Object {
       thisStoresPatient: this.thisStoresPatient,
       nationality: this.nationality?.toJSON(),
       ethnicity: this.ethnicity?.toJSON(),
+      nameNotes: this.nameNotes?.map(nameNote => nameNote.toObject()),
```

Comment:
This is so the names from realm convert to an object which more closely resembles the ones sent from the patient lookup api